File: javascript/7kyu/Minimize Sum Of Array Array Series 1.js
// https://www.codewars.com/kata/5a523566b3bfa84c2e00010b

// Given an array of integers , Find the minimum sum which is obtained from summing each Two integers product.

function minSum(arr) {
    arr.sort((x,y)=>x-y)
    s=0
    while(arr.length)s+=arr.pop()*arr.shift()
    return s
}

File: java/7kyu/Minimize Sum Of Array Array Series 1.java
// Minimize Sum Of Array (Array Series #1)

// Given an array of integers , Find the minimum sum which is obtained from summing each Two integers product .

public class Solution
{
    public static int minSum(int[] numbers)
    {
        java.util.Arrays.sort(numbers);
        int min = 0;
        
        for (int i = 0; i < numbers.length / 2; i++)
            min += numbers[i] * numbers[numbers.length - 1 - i];
        
        return min;
    }
}